### PR TITLE
Add web3.js-eea reference doc  

### PR DESCRIPTION
**Pull request by MadelineMurray**
Add web3.js-eea reference doc

### CODE REVIEW

File: docs/Reference/web3js-eea-Methods.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+The Options parameter is used by: 
+
+* [`generatePrivacyGroup`](#generateprivacygroup)
+* [`getTransactionCount`](#gettransactioncount)
+* [`sendRawParameter`](#sendrawtransaction)
+
+The Options parameter has the following properties: 
+
+* `privateKey`: Ethereum private key with which to sign the transaction
+* `privateFrom` : Orion public key
```
Comment (iikirilov) [inline docs/Reference/web3js-eea-Methods.md]:
```suggestion
* `privateFrom` : Orion public key of sender
```

File: docs/Reference/web3js-eea-Methods.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
+description: web3js-eea methods reference
+<!--- END of page meta data -->
+
+Use the [web3.js-eea library](https://github.com/PegaSysEng/eeajs) to [create and send 
+private transactions](../Privacy/Private-Transactions/eeajs.md).
+
+## Options Parameter 
+
+The Options parameter is used by:
```
Comment (bgravenorst) [inline docs/Reference/web3js-eea-Methods.md]: Is "Options" the actual parameter name? If so, should it be formatted?

File: docs/Reference/web3js-eea-Methods.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+## getMarkerTransaction
+
+Gets the [privacy marker transaction](../Privacy/Private-Transaction-Processing.md) transaction receipt.
+
+**Parameters**
+
+`txHash` - `string` : Transaction hash of the private transaction
+
+`retries` - `int` : Number of attempts to make to get the private marker transaction receipt
```
Comment (bgravenorst) [inline docs/Reference/web3js-eea-Methods.md]: Maximum number of attempts to get the private marker transaction receipt

File: docs/Reference/web3js-eea-Methods.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+Gets the private transaction receipt using [`eea_getTransactionReceipt`](../Reference/Pantheon-API-Methods.md#eea_gettransactionreceipt).
+
+**Parameters**
+
+`txHash` - `string` : Transaction hash of the private transaction
+
+`enclavePublicKey` - `string` : [`privateFrom` key for the transaction](#options-parameter) 
+
+`retries` - `int` : Optional. Number of attempts to make to get the private marker transaction receipt. Default is `300`.
```
Comment (bgravenorst) [inline docs/Reference/web3js-eea-Methods.md]: Maximum number of attempts...

File: docs/Reference/web3js-eea-Methods.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+* [`getTransactionCount`](#gettransactioncount)
+* [`sendRawParameter`](#sendrawtransaction)
+
+The Options parameter has the following properties: 
+
+* `privateKey`: Ethereum private key with which to sign the transaction
+* `privateFrom` : Orion public key
+* `privateFor` : Orion public keys of recipients
+* `nonce` : Optional. If not provided, calculated using [`eea_getTransctionCount`](../Reference/Pantheon-API-Methods.md).
+* `to` : Optional. Contract address to which to send the transaction. Do not specify for contract deployment transactions
```
Comment (bgravenorst) [inline docs/Reference/web3js-eea-Methods.md]: Contract address to send the transaction to.

File: docs/Reference/web3js-eea-Methods.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+**Parameters**
+
+`txHash` - `string` : Transaction hash of the private transaction
+
+`enclavePublicKey` - `string` : [`privateFrom` key for the transaction](#options-parameter) 
+
+`retries` - `int` : Optional. Number of attempts to make to get the private marker transaction receipt. Default is `300`. 
+
+`delay` - `int` : Optional. Delay between retries in milliseconds. Optional. Default is `1000`.
```
Comment (bgravenorst) [inline docs/Reference/web3js-eea-Methods.md]: second "Optional" is probably redundant

File: docs/Reference/web3js-eea-Methods.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Reference/web3js-eea-Methods.md]: Fixed

File: docs/Reference/web3js-eea-Methods.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/web3js-eea-Methods.md]: No - not an actual parameter

File: docs/Reference/web3js-eea-Methods.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Reference/web3js-eea-Methods.md]: Fixed

File: docs/Reference/web3js-eea-Methods.md (hunk 4 above)
Comment (MadelineMurray) [reply docs/Reference/web3js-eea-Methods.md]: Fixed

File: docs/Reference/web3js-eea-Methods.md (hunk 5 above)
Comment (MadelineMurray) [reply docs/Reference/web3js-eea-Methods.md]: Fixed

File: docs/Reference/web3js-eea-Methods.md (hunk 6 above)
Comment (MadelineMurray) [reply docs/Reference/web3js-eea-Methods.md]: Fixed

**Pull request merged**